eng/init-source-only.proj

```diff
@@ -15,6 +15,7 @@
 
   <PropertyGroup>
     <ExternalTarballsDir>$([MSBuild]::NormalizeDirectory('$(PrereqsPackagesDir)', 'archive'))</ExternalTarballsDir>
+    <ExternalSharedComponentsTarballsDir>$([MSBuild]::NormalizeDirectory('$(PrereqsPackagesDir)', 'archive-shared-components'))</ExternalSharedComponentsTarballsDir>
```

Comment:
Why does this need to be placed in a separate directory?  Can't a common archive directory be used?  In my personal workflow, I like to preserve this dir while cleaning so that I don't have to re-pull it as part of prep.  Having one dir to preserve is nice versus multiple.

Reply:
That's fair. I'll probably just integrate the prebuilt directory in that as well.

Reply:
Actually, I won't change the `prebuilt` directory; I'll leave it where it is. That's because it doesn't represent input to the build. Rather, it will contain the extracted contents of the prebuilts tarball. In reality, it shouldn't exist at all in source. I'll leave this to https://github.com/dotnet/source-build/issues/5272.